app/imports/components/comments/Comments.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+import { Comment } from '/imports/components/comments/Comment.jsx';
+
+export const Comments = ({comments, commentsLimit, postId}) => {
+
+  if (comments === undefined) {
+    return null;
+  }
+
+  let commentsRows = [];
+  let moreComments = null;
+
+  if (commentsLimit === undefined ) {
```

Comment:
I think that is a bad pattern for building site setting functionality. It isn't about global vars to save remembering but about being able to tweak the site without getting into any code